src/patternfly/components/Card/card.scss

```diff
@@ -46,11 +46,26 @@
   --pf-c-card__actions--PaddingLeft: var(--pf-global--spacer--md);
   --pf-c-card__actions--child--MarginLeft: var(--pf-global--spacer--sm);
 
+  // Toggle right
+  --pf-c-card__header--m-toggle-right--toggle--MarginRight: calc(var(--pf-global--spacer--form-element) * -1);
+  --pf-c-card__header--m-toggle-right--toggle--MarginLeft: 0;
+
+  // nested card
+  --pf-c-card--c-card--BoxShadow: none;
+
+  // toggle right
+  --pf-c-card__header--m-toggle-right-toggle--MarginRight: calc(var(--pf-global--spacer--md) * -1);
+  --pf-c-card__header--m-toggle-right-toggle--MarginLeft: var(--pf-global--spacer--xs);
+
   display: flex;
   flex-direction: column;
   background-color: var(--pf-c-card--BackgroundColor);
   box-shadow: var(--pf-c-card--BoxShadow);
 
+  & > & {
+    --pf-c-card--BoxShadow: var(--pf-c-card--c-card--BoxShadow);
```

Comment:
It feels a little specific and unexpected that a card as a direct child of a card doesn't have a box-shadow. I'm thinking of how a user would know that is the case without documentation. Although if this is a common usage of the card we want to support, we could add an example to the cards example page. 

Alternatively to be more modular we could add a `pf-m-no-box-shadow` variation of the card, which would allow a user to nest a card in something else other than being the direct descendent and not have a shadow.